glean-core/ffi/src/boolean.rs

```diff
@@ -4,29 +4,25 @@
 
 use ffi_support::FfiStr;
 
-use crate::{define_metric, handlemap_ext::HandleMapExtension, GLEAN};
+use crate::{define_metric, handlemap_ext::HandleMapExtension, with_glean_value};
```

Comment:
nit: This function name didn't immediately make sense to me.  Maybe `with_glean_singleton`?